Topic-1-HTML/1_Ba.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <style>
+        table {
+          font-family: arial, sans-serif;
+          border-collapse: collapse;
+          width: 100%;
+        }
+        
+        td, th {
+          border: 1px solid #dddddd;
+          text-align: left;
+          padding: 8px;
+        }
+
+        #saving{
+            color: #218b26;
+        }
+
+        #spending{
+            color: #ff0000; 
+        }
+        
+    </style>  
+</head>
+<body>
+    <table>
+        <tr>
+            <th>Saving</th>
+            <th>Spending</th>
+            <th>Final</th>
+        </tr>
+        <tr>
+            <td id="saving">2000</td>
```

Comment:
use `class` instead of `id`

Reply:
I consider this more a suggestion than a mandatory change for this exercise. In practice `ids` aren't used for styling